File: app/src/main/java/com/incite/moviequiz/SplashActivity.java
package com.incite.moviequiz;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.widget.ScrollView;

import com.airbnb.lottie.LottieAnimationView;

public class SplashActivity extends AppCompatActivity {

    LottieAnimationView splashAnimation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        splashAnimation = findViewById(R.id.splash_anim);
        splashAnimation.setSpeed(1.2f);
        splashAnimation.playAnimation();




        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent i = new Intent(SplashActivity.this, MenuActivity.class);
                startActivity(i);
                finish();
            }
        },300); //2600 def
    }
}

File: app/src/main/java/com/incite/moviequiz/presentation/splash/SplashActivity.kt
package com.incite.moviequiz.presentation.splash

import androidx.appcompat.app.AppCompatActivity
import com.airbnb.lottie.LottieAnimationView
import android.os.Bundle
import android.content.Intent
import android.os.Handler
import com.incite.moviequiz.presentation.menu.MenuActivity
import com.incite.moviequiz.R

class SplashActivity : AppCompatActivity() {
    private lateinit var splashAnimation: LottieAnimationView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        splashAnimation = findViewById(R.id.splash_anim)
        splashAnimation.speed = 1.2f
        splashAnimation.playAnimation()

        Handler().postDelayed({
            val i = Intent(this, MenuActivity::class.java)
            startActivity(i)
            finish()
        }, 300) //2600 def
    }
}